core/src/test/java/uk/gov/gchq/koryphe/impl/predicate/AreInTest.java

```diff
@@ -166,4 +166,26 @@ protected Iterable<AreIn> getDifferentInstancesOrNull() {
                 new AreIn(VALUE2)
         );
     }
+
+    @Test
+    public void shouldRejectWhenValuesIsEmptyAndEmptyNotAllowed() {
```

Comment:
Maybe more tests with null being used as well as empty list would be good

Reply:
Ok. Will add that